modules/mod_translation/support/translation_po.erl

```diff
@@ -26,10 +26,11 @@
 
 
 %% @doc Take the list of found labels per module and generate all po files for those labels in the module directories.
```

Comment:
Line 28 is too long: %% @doc Take the list of found labels per module and generate all po files for those labels in the module directories..